Code/classy_hrm.py

```diff
@@ -0,0 +1,19 @@
+def classy_hrm(file='ecg_data1.csv', window=20, brady_max=40, tachy_min=100):
```

Comment:
Need units on the input parameters.

Reply:
Units added to doc string for classy_hrm